Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/SubmissionResultPositiveOtherWarningFragment.kt

```diff
@@ -187,6 +187,7 @@ class SubmissionResultPositiveOtherWarningFragment : Fragment(),
         if (keys.isNotEmpty()) {
             submissionViewModel.submitDiagnosisKeys(keys)
         } else {
+            SubmissionService.submissionSuccessful()
```

Comment:
Nitpick: From a separation of concerns perspective i would prefer it if the SubmissionService was not directly included in the Fragment but called from the ViewModel instead.

Reply:
Hi Kolya, since we do not send any network request, the current way seemed like the most efficient and straight forward.

@Fabian-K  what do you think of this?

Reply:
My main concern here is not with networking but with DI, if we later add support for DI this fragment would turn into an additional injection point for the `SubmissionService`, which could be avoided if we only use the `SubmissionService` inside of the `SubmissionViewModel`

Reply:
@kolyaopahle changes have been made, please take a look.

Thank you